webpack.config.js

```diff
@@ -30,6 +30,13 @@ const hubspotConfig = ({ portal, autoupload } = {}) => {
           use: [
             MiniCssExtractPlugin.loader,
             { loader: 'css-loader', options: { url: false } },
+            {
+              loader: 'postcss-loader',
+              options: {
+                plugins: () => [require('autoprefixer')]
```

Comment:
I think it would be best if the require is at the top of the file instead of being required inline. It makes it clearer what the dependencies are when looking at a file.